simplecpp.cpp

```diff
@@ -1716,66 +1716,123 @@ namespace simplecpp {
         return ostr.str();
     }
 #else
-#define realFilename(f)  f
+    // Boring implementation, but could help in platform-independent tests:
+    std::string realFilename(const std::string &f)
```

Comment:
it would be nice to test realFilename in windows. since there has been problems. but I am skeptic about testing it with unit tests since it needs files on the disk.

I don't say you should implement that now... but maybe a separate test script for windows could test it.